frontend/src/features/apartment/ApartmentCreatePage.tsx

```diff
@@ -170,18 +172,18 @@ const ApartmentCreatePage = () => {
                             setFormData={setFormData}
                             error={error}
                         />
-                    </div>
-                    <div className={"row"}>
                         <FormInputField
                             inputType="number"
                             fractionDigits={2}
                             label="Pinta-ala"
                             fieldPath="surface_area"
-                            required
+                            required={true}
```

Comment:
These two are equivalent :)

Reply:
..also, should be fixed in a later commit. Or at least I couldn't find a `required={true}` in `ApartmentCreatePage.tsx` anymore.